dev-docs/adding-a-new-page-to-the-installer.md

```diff
@@ -0,0 +1,55 @@
+# Adding a new page to the installer
+
+## Add your page
+- Under `apps/ubuntu_bootstrap/lib/pages`, create a new directory for the page you would like to add
+ie: `apps/ubuntu_bootstrap/lib/pages/<your-feature>`
```

Comment:
Depending on how general the page is and whether we want to use it in `ubuntu_init` or elsewhere later on, the page could be added to `ubuntu_provision` instead